kitsune/sumo/form_fields.py

```diff
@@ -83,6 +83,41 @@ def to_python(self, value):
         return users
 
 
+class MultiUsernameOrGroupnameField(forms.Field):
+    """Form field that takes a comma-separated list of usernames or groupnames
+    and validates that users/groups exist for each one, and returns the list of
+    users/gorups."""
+
+    def to_python(self, value):
+        if not value:
+            if self.required:
+                raise forms.ValidationError(_("To field is required."))
+            else:
+                return []
+
+        users_and_groups = []
+        for user_or_group in value.split(","):
+            if ":" in user_or_group:
```

Comment:
nit: you can skip the `if` check by using
```python
_type, *names = user_or_group.split(":")
```

Reply:
Altered functionality a good bit - this is no longer relevant.